syntax = "proto3";

service LoginCall {
  rpc Login(LoginRequest) returns(LoginReply){}
}

message LoginRequest {
  string username = 1;
  string password = 2;
  optional string body = 3;
}

message LoginReply {
  optional string object = 1;
  optional string error = 2;
  int32 status_code = 3;
}
